src/sparql/ordo-add-annotation-based-mappings.ru.jinja2

```diff
@@ -0,0 +1,29 @@
+prefix skos: <http://www.w3.org/2004/02/skos/core#>
+prefix IAO: <http://purl.obolibrary.org/obo/IAO_>
+prefix ECO: <http://purl.obolibrary.org/obo/ECO_>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix oio: <http://www.geneontology.org/formats/oboInOwl#>
+prefix def: <http://purl.obolibrary.org/obo/IAO_0000115>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+prefix obo: <http://purl.obolibrary.org/obo/>
+prefix oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
+
+
+INSERT {
```

Comment:
We want to delete the old pattern as well as adding the new one:

```
DELETE {
?cls ?mapping ?value .

  ?xref_anno a owl:Axiom ;
    owl:annotatedSource ?cls ;
    owl:annotatedProperty ?mapping ;
    owl:annotatedTarget ?value ;
    ECO:0000218 ?mapping_precision_string .
}
```

Reply:
Yep, that sounds good. Thanks for providing the `DELETE` clause!